packages/vite-plugin-cloudflare/playground/vite-module-to-test.ts

```diff
@@ -0,0 +1,63 @@
+import * as rolldownVite from "rolldown-vite";
+import * as defaultVite from "vite";
+import * as vite6 from "vite-6";
+import * as vite7 from "vite-7";
```

Comment:
having a separate/specific `vite7` is unnecessary and I could just use `defaultVite` for testing vite 7, I just included this because I think that clearly separating the default vite module and the vite 7 one makes things clearer, I am happy to just reuse `defaultVite` if people don't like this